config/vite/vite.config.mts

```diff
@@ -0,0 +1,29 @@
+/// <reference types="vitest" />
+
+import { defineConfig } from 'vite'; // eslint-disable-line import/no-extraneous-dependencies -- this is a dev-only dependency
+import { fileURLToPath } from 'node:url';
+
+// eslint-disable-next-line import/no-default-export -- default export is a Vite requirement
+export default defineConfig({
+  build: {
+    emptyOutDir: true,
+    lib: {
+      entry: fileURLToPath(new URL('/src/RollbarClient.mts', import.meta.url)),
+      fileName: 'RollbarClient',
+      formats: ['es'],
+    },
+    minify: 'esbuild',
```

Comment:
`esbuild` is the underlying tool written in Go that makes Vite so fast. I'm leveraging it as much as possible to speed up builds and minifies.